js/components/widgets/image-picker.vue

```diff
@@ -129,9 +128,9 @@ export default {
         }
     },
     methods: {
-        save: function() {
+        save() {
             if (this.HAS_FILE_API) {
-                let data = {};
+                const data = {};
```

Comment:
I'm only a little puzzle by this as it is declared as a constant though will be treated as a viariable object (`data.bbox =`). I know JS does not throw an error yet but I'd rather keep `let` in this case.